packages/env/lib/config/test/__snapshots__/config.js.snap

```diff
@@ -14,10 +14,10 @@ exports[`readConfig config file should match snapshot 1`] = `
         "TEST_VAL3": false,
         "WP_DEBUG": true,
         "WP_ENVIRONMENT_TYPE": "local",
-        "WP_HOME": "http://localhost:2000/",
```

Comment:
As far as I can tell, we aren't supposed to put trailing slashes on these constants. [They're actually removed automatically by WordPress.](https://github.com/WordPress/wordpress-develop/blob/b24f990f4f7789cee65628ef090a9e68a2798b5b/src/wp-admin/includes/upgrade.php#L2652). Since my new port addition function doesn't add a trailing slash, we don't have one here, but that's okay.